packages/mobx/__tests__/v5/base/make-observable.ts

```diff
@@ -1409,3 +1410,16 @@ test("@override must override", () => {
         /^\[MobX\] 'Child\.prototype\.action' is decorated with 'override', but no such decorated member was found on prototype\./
     )
 })
+
+test("makeAutoObservable + production build #2751", () => {
+    const mobx = require(`../../../dist/mobx.cjs.production.min.js`)
```

Comment:
Why we need require `mobx` from `dist` ?
```bash
 ✓ makeAutoObservable + production build #2751 (10 ms)
```
Btw I write the unit test, passed in current `main` branch.
So this unit test is test for what code change ? Looks like it's unuseful...  : (